dlt/common/time.py

```diff
@@ -142,6 +141,8 @@ def ensure_pendulum_time(value: Union[str, datetime.time]) -> pendulum.Time:
         result = parse_iso_like_datetime(value)
         if isinstance(result, pendulum.Time):
             return result
+        elif isinstance(result, pendulum.DateTime):
```

Comment:
note: not sure what is going on here, I will have to investigate and see if clickhouse just saves times as datetime or wether there is something else going on here.